proxy/envoy/testdata/envoy-v0.json.golden

```diff
@@ -1,7 +1,53 @@
 {
   "listeners": [
     {
-      "address": "tcp://0.0.0.0:1081",
+      "address": "tcp://0.0.0.0:80",
```

Comment:
In the golden files, can we add comment lines describes each type of listener and filter? It is hard to understand otherwise what each listener and filter does.